components/prism-cmake.js

```diff
@@ -0,0 +1,18 @@
+	'boolean': /\b(ON|OFF|TRUE|FALSE)\b/,
```

Comment:
Please only use non-capturing groups, so `\b(?:ON|...)\b` instead of  `\b(ON|...)\b`.

This also applied for all other tokens.

Reply:
Of course, but not to comments, right?

Reply:
pushed

Reply:
Comments?

Reply:
`	'comment': /#.*/,` Should this be a non-capturing group?

Reply:
Well, it's not a group, so probably not :)

Reply:
No, we don't need to introduce an unnecessary group, so you can leave it as is.